lib/jxl/modular/transform/squeeze.cc

```diff
@@ -43,36 +96,109 @@ Status InvHSqueeze(Image &input, uint32_t c, uint32_t rc, ThreadPool *pool) {
     return true;
   }
 
+#if HWY_TARGET != HWY_SCALAR
+
+  // somewhat complicated trickery just to be able to SIMD this
```

Comment:
Can you expand on what is that trickery? :)

Reply:
Added a comment that attempts to explain it.